src/utils/dev.ts

```diff
@@ -207,7 +207,9 @@ class NuxtDevServer extends EventEmitter {
     }
 
     // Remove websocket handlers on close
-    this._currentNuxt.hooks.hookOnce('close', () => {
+    // nuxt bridge doesn't have support for `hookOnce` here
+    const unsubClose = this._currentNuxt.hooks.hook('close', () => {
+      unsubClose()
```

Comment:
With https://github.com/nuxt/bridge/pull/921, I think we can revert this change and just keep the reordered `ready()` call.

```suggestion
    this._currentNuxt.hooks.hookOnce('close', () => {
```